frontend/public/components/_dropdown.scss

```diff
@@ -255,3 +255,7 @@ $dropdown-background--hover: $color-pf-black-200; // pf-c-dropdown__menu-item--h
     text-overflow: ellipsis;
   }
 }
+
+button.dropdown__inline-error {
+  box-shadow: 0px 0.5px 2px 1px red;
+}
```

Comment:
This is not aligned with any of PF designs. I'm even ok without having the style match PF right now:
![image](https://user-images.githubusercontent.com/14068621/69301402-36ea5f80-0be4-11ea-89db-9940275bca38.png)